config/presets/fastnet.go

```diff
@@ -50,6 +50,9 @@ func fastnet() config.Config {
 	conf.POST.MaxNumUnits = 4
 	conf.POST.MinNumUnits = 2
 
+	conf.POET.CycleGap = 10 * time.Second
+	conf.POET.PhaseShift = 20 * time.Second
```

Comment:
they should be set in systest if you want to set them

Reply:
done